### PR TITLE
Setup yarn and add some docs

### PR DESCRIPTION
**Pull request by somewhatabstract**
## Summary:

Issue: FEI-4001

## Test plan:
Nothing for this bit

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,104 +1,7 @@
...
-
-# FuseBox cache
-.fusebox/
-
-# DynamoDB Local files
-.dynamodb/
-
-# TernJS port file
-.tern-port
+!.vscode
```
Comment (kevinbarabash) [inline .gitignore:5]: Does git ignore this directory by default?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+You can now work on `wonder-stuff`. We prefer [🔗Visual Studio Code](https://code.visualstudio.com/) as our development environment (it's cross-platform and awesome), but please use what you feel comfortable with (we'll even forgive you for using vim).
+
+### 🧪 Code Quality
+
+#### Manual
+
+We love code reviews. If there are open pull requests, please feel free to review them and provide feedback. Feedback is a gift and code reviews are often a bottleneck in getting new things released. Jump in, even if you don't know anything; you probably know more than you think.
+
+💭**REMEMBER** Be kind and considerate. Folks are volunteering their time and code reviews are a moment of vulnerability where a criticism of the code can easily become a criticism of the individual that wrote it.
```
Comment (kevinbarabash) [inline CONTRIBUTING.md:39]: This is very welcoming.  Thanks for including this.  It does feel a little at odds with the "We are not currently accepting externally..." lines above.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+📖 Be sure to read our [Code of Conduct](https://github.com/Khan/render-gateway/blob/master/CODE_OF_CONDUCT.md).
+
+## 🛑 Bugs And Feature Requests
+
+⚠️ **We are not currently accepting externally raised bugs and feature requests**
+
+## 💻 Code Changes
+
+⚠️ **We are not currently accepting externally provided code changes**
```
Comment (kevinbarabash) [inline CONTRIBUTING.md:16]: Now that we're using GitHub internally for code reviews.  How do we differentiate code reviews from KA employees vs. external people that try to create code reviews?  I wonder if there's maybe some way that we can leverage Gerald to verify that a PR is coming from a KA employee and auto-close ones that don't.  No action req'd.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "wonder-stuff",
+  "version": "1.0.0",
```
Comment (kevinbarabash) [inline package.json:3]: Starting out at 1.0.0. feels a bit premature.

File: package.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+  "description": "Packages for sharing features across JavaScript-based projects",
+  "main": "index.js",
```
Comment (kevinbarabash) [inline package.json:5]: Does it make sense to specify `main` if we're going to be publishing multiple packages?

**Review by kevinbarabash — APPROVED** [review]
LGTM.  A few minor nits, but nothing blocking.  Thanks for setting up this new repo.

File: package.json (hunk 5 above)
Comment (somewhatabstract) [reply package.json:5]: Just copied wonder blocks

**Pull request merged**